### PR TITLE
Bugfix/broken metanorma config subcommand

### PR DESCRIPTION
**Pull request by CAMOBAP**
- https://github.com/metanorma/mn-samples-bipm/pull/30

Kind of urgent because of this PR fix:
 - `metanorma config` commad
 - fix parsing font-related args to `metanorma site generate`

### CODE REVIEW

**Comment by ronaldtse** [comment]
Build failing right now but feel free to merge when done.

File: spec/metanorma/cli/collection_spec.rb [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-    expect(File.exist?("spec/results/dummy.html")).to be true
-    expect(File.exist?("spec/results/rice-amd.final.html")).to be true
-    expect(File.exist?("spec/results/rice-en.final.html")).to be true
-    expect(File.exist?("spec/results/rice1-en.final.html")).to be true
-    FileUtils.rm_rf of
+    expect(File.exist?("#{RESULTS}/index.html")).to be true
+    expect(File.exist?("#{RESULTS}/dummy.html")).to be true
+    expect(File.exist?("#{RESULTS}/rice-amd.final.html")).to be true
+    expect(File.exist?("#{RESULTS}/rice-en.final.html")).to be true
+    expect(File.exist?("#{RESULTS}/rice1-en.final.html")).to be true
```
Comment (abunashir) [inline spec/metanorma/cli/collection_spec.rb:19]: Looks like we are doing the same thing in couple of places, what do you think about having some helper method here? maybe something like

```ruby
#usages:
expect(spec_result_file_exists?("rice-en.final.html")).to be true

# helper method
def spec_result_file_exists?(filename)
  file_path = Pathname.new(RESULTS).join(filename)
  File.exist?(file_path)
end
```

File: spec/metanorma/cli/collection_spec.rb [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
```
Comment (abunashir) [inline spec/metanorma/cli/collection_spec.rb:30]: Same here

**Review by abunashir — COMMENTED** [review]
Apart from couple of nitpicking, it looks good. Once the CI issue is fixed then please feel free to merge.

File: spec/metanorma/cli/collection_spec.rb (hunk 1 above)
Comment (CAMOBAP) [reply spec/metanorma/cli/collection_spec.rb]: Done

File: spec/metanorma/cli/collection_spec.rb (hunk 2 above)
Comment (CAMOBAP) [reply spec/metanorma/cli/collection_spec.rb]: Done

**Review by abunashir — APPROVED** [review]
Looks good to me 👍🏼

File: spec/metanorma/cli/generator_spec.rb [hunk 3]
```diff
@@ -71,35 +63,30 @@
...
-            document,
-            type: "csd",
-            overwrite: true,
-            doctype: "standard",
-            template: template,
-          )
-        }
+        Metanorma::Cli::Generator.run(
+          document,
+          type: "csd",
```
Comment (ronaldtse) [inline spec/metanorma/cli/generator_spec.rb:68]: @CAMOBAP haven't we changed to `cc` already?

**Review by ronaldtse — APPROVED** [review]
Thanks @CAMOBAP ! When @abunashir reviews with a pass, it's likely a pass :wink:

**Comment by abunashir** [comment]
I'm the gatekeeper 😉😄, fun aside - he's doing a good job!

> Thanks @CAMOBAP ! When @abunashir reviews with a pass, it's likely a pass 😉

**Pull request merged**